src/utils/api.ts

```diff
@@ -34,3 +34,57 @@ export const searchOrders = async (
   const response = await axios.request(request);
   return response.data;
 };
+
+export const getCountries = async (
+  authData: TAuthData,
+  data: { pageNo: number; pageSize: number }
+): Promise<unknown> => {
+  const { server, token } = authData;
+  const { pageNo, pageSize } = data;
+  const request: AxiosRequestConfig = {
+    url: `${server}/api/v1/countries?page_no=${pageNo}&page_size=${pageSize}`,
+    method: "GET",
+    headers: {
+      Authorization: token,
+    },
+  };
+
+  const response = await axios.request(request);
+  return response.data;
+};
+
+export const getCountryDetailsByCountry = async (
+  authData: TAuthData,
+  data: { symbol: string }
+): Promise<unknown> => {
```

Comment:
Use an Axios instance with a base URL and add type safety for API responses (e.g., ```axios.get<Type>```).